README.md

```diff
@@ -0,0 +1,2 @@
+#Coastal Ocean Assessment Tool
+This is a package for working on coastal assessment of NEMO data
```

Comment:
Should NEMO be a link to somewhere with more info? https://www.seadatanet.org/Software/NEMO for instance?

Reply:
good questions - will ask jeff/ others and resolve here